core/template.php

```diff
@@ -474,7 +475,7 @@ function tpl_show($video)
     }
 
     // add episodes information
-    if (is_array($video['episodes']))
+    if (array_key_exists( 'episodes', $video ))
```

Comment:
Same here, functionality changed, should be something like:
`if (array_key_exists('episodes', $video) && is_array($video['episodes']))`
to keep current functionality and negate warning

Reply:
code amended